platform/linux-generic/example/ml/conv_gen.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
With python shebang (and executable rights), this wouldn't need `python3 conv_gen.py`.

Reply:
There are other similar scripts, which should all be changed if we want to change them. I'm leaving this as-is.